pillar/top.sls

```diff
@@ -149,8 +149,8 @@ base:
   'G@roles:devstack and P@environment:dev':
     - match: compound
     - devstack
-    - consul_devstack
-    - mysql_devstack
-    - rabbitmq_devstack
-    - elasticsearch_devstack
-    - mongodb_devstack
+    - consul.devstack
+    - db.mongodb_devstack
+    - db.mysql_devstack
```

Comment:
I would just consider moving the mongo and mysql pillar files under directories for their respective services and factoring out any common values for Mongo into an `init.sls` file.